### PR TITLE
Fix docs/Concepts.md links

### PR DESCRIPTION
**Pull request by vdemeester**
… fixing my `prow` mistakes 😅 Follow-up for #194 🙇

/cc @bobcatfish 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: docs/Concepts.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ For example:
 * A task's input could be a github source which contains your application code.
 * A task's output can be your application container image which can be then deployed in a cluster.
 
-Read more on PipelineResources and their types [here](./docs/pipeline-resources.md)
+Read more on PipelineResources and their types [here](./pipeline-resources.md)
```
Comment (pivotal-nader-ziada) [inline docs/Concepts.md]: this link is still broken because the resources file is deleted, should be pointing to `using.md`

Comment (vdemeester) [reply docs/Concepts.md]: oh good point :sweat_smile:

Comment (vdemeester) [reply docs/Concepts.md]: Updated :wink:

**Comment by bobcatfish** [comment]
Thanks for taking care of these for me @vdemeester !! :D :heart:

**Pull request merged**